include/circt/Dialect/SV/SVPasses.td

```diff
@@ -24,6 +24,11 @@ def HWCleanup : Pass<"hw-cleanup", "hw::HWModuleOp"> {
       opportunities for other simpler passes (like canonicalize).
   }];
 
+  let options = [
+    Option<"emitSeparateAlwaysBlocks", "emit-separate-always-blocks", "bool",
+           "false", "Emit assigments to registers in separate always blocks">
+  ];
+
```

Comment:
For the HWCleanup pass this option is more like "don't merge always blocks".  This option sounds like it will split already formed always blocks.